matchmaker/views/matchmaker_api_2_3_tests.py

```diff
@@ -590,7 +592,7 @@ def test_update_mme_submission(self, mock_mme_nodes):
         })
 
         self.assertDictEqual(response_json['mmeSubmissionsByGuid'], {new_submission_guid: {
-            'mmeResultGuids': response_json['mmeResultsByGuid'].keys(),
+            'mmeResultGuids': sorted(list(response_json['mmeResultsByGuid'].keys()), reverse = True),
```

Comment:
Using `sorted()` to make the order of the keys to be determined. So that the expected result can match the real result.